registry/static/html/mods_find.html

```diff
@@ -33,15 +33,17 @@ <h3>find modules</h3>
                 no results
             </span>
         {{else}}
+        <form method="GET" id="populate-new" action="/mods/new">
```

Comment:
I'm open to changing this to a POST

Reply:
Let's do that for the technical correct-ness. (no concept of a request body exists for GET requests, even if most browsers let it work).

Reply:
currently there's no request body. It just puts all the params into the url. This could get messy if someone selected a large number of packages (there's a browser-side limit on the length of the url), and it's also ugly. I feel weird about making this a POST because it doesn't actually change any state.
I don't feel strongly about this, just wanted to make sure you understood the dilemma. Shall I still go ahead and make this a POST?